cards/faq-accordion/component.js

```diff
@@ -64,7 +64,16 @@ class faq_accordionCardComponent extends BaseCard['faq-accordion'] {
 
     const contentEl = this._container.querySelector(accordionContentSelector);
     let isExpanded = this._container.querySelector(`.${accordionExpandedClass}`);
-    contentEl.style.height = `${isExpanded ? contentEl.scrollHeight : 0}px`;
+
+    let scrollHeight = contentEl.scrollHeight;
+    const parentEl = document.querySelector('.yxt-Answers-component--unmounted');
```

Comment:
yes, it's from the vertical results template in the SDK

Reply:
would it be possible to update the hbs template instead? I think that's typically what we edit when we want a change like this. I'm a little worried adding and removing the css class will cause a small flash (also it's a little spaghetti to add it and instantly remove it)

Reply:
do you mean remove the class from the hbs template? because I thought this class was needed in the template to make sure the card is not displayed until it's finished mounting?

Reply:
oh I see this is for Connor's progressive rendering thing sorry had a brainfart. Could we just set the height to 100% or something for the initial load?

Reply:
the transitions don't work properly when going from height auto or 100% to a pixel value (or vice versa)

Reply:
I see, could we limit this to just the initial load somehow then?

Reply:
I think we would want this to happen whenever the component mounts, right? because when it's mounting, we won't be able to access scrollHeight without adding and removing the css class. the actual toggling of the expansion is handled by the click listener and doesn't use this logic